test/Microsoft.IdentityModel.Tests/IdentityComparer.cs

```diff
@@ -86,6 +86,7 @@ public class IdentityComparer
                 { typeof(OpenIdConnectMessage).ToString(), CompareAllPublicProperties },
                 { typeof(Reference).ToString(), CompareAllPublicProperties },
                 { typeof(RsaSecurityKey).ToString(), CompareAllPublicProperties },
+                { typeof(RSAParameters).ToString(), AreRsaParametersEqual },
```

Comment:
With this here, could we remove: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/rsa/test/Microsoft.IdentityModel.Tests/IdentityComparer.cs#L652

Reply:
I think you're right! This doesn't seem to be necessary anymore; I just removed it.

Reply:
Can you add a test to IdentityComparerTests to cover RSAParameters.
Somewhere in here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev/test/Microsoft.IdentityModel.Tokens.Tests/IdentityComparerTests.cs#L370

Reply:
Just added in a separate test for RSAParameters.